### DISCUSSION TITLE
please make this too

### DISCUSSION THREAD

**Discussion by ghost** [obrobrio2000/Samsung-Quick-Share-4-All#2] [General]
I am using Samsung-Quick-Share-4-All program very well. thank you.

Could you please make this(https://www.microsoft.com/en-us/p/bixby/9phbjm786kwx?activetab=pivot:overviewtab) too?

Thanks for reading.

**Comment by magliocchetti** [comment]
It's not my first priority because I personally don't use Bixby but as soon as I have some time I'll look into it :)
Thanks for sharing and I'm glad Samsung-Quick-Share-4-All is working well for you :)
